js/components/controlBar.js

```diff
@@ -219,6 +221,56 @@ var ControlBar = React.createClass({
     this.props.controller.state.focusedControl = null;
   },
 
+  /**
+   * Will handle the keydown event when the controlBar is active and it will restrict
+   * tab navigation to elements that are within it when the player is in fullscreen mode.
+   * @private
+   * @param {Object} evt Keydown event object.
+   */
+  handleControlBarKeyDown: function(evt) {
+    if (
+      evt.key !== CONSTANTS.KEY_VALUES.TAB
```

Comment:
is evt guaranteed to not be null?  Someone may end up calling this outside of a callback.

Reply:
Now that I look at the rest of the code, it's very dependent on evt. So it's probably ok not having the check.

Reply:
Yes this can only work as an event handler. I think it should just crash if called directly since it's not its intended use